resources/dicts/thoughts/alive/apprentice.json

```diff
@@ -263,12 +258,35 @@
             "Is hoping to be sent to battle",
             "Is picking a fight with r_c",
             "Is bored during a hunting patrol",
+            "Is busy mastering a fighting technique",
             "Is in trouble after scratching r_c during an argument"
         ],
         "main_trait_constraint": [
             "fierce"
         ]
     },
+    {
+        "id": "competitive_warrior_apprentice",
```

Comment:
These are new thoughts, rather than bug fixes? If so we might want to wait until after the next release to add them.